tests/plugins/wysiwygarea/manual/detachedtwoeditors.html

```diff
@@ -0,0 +1,38 @@
+<button id="toggleButton">Toggle</button>
```

Comment:
I'd move this button directly above detachable editor. It seems more intuitive for me. Additionally, currently I need to scroll page up to use the button after interacting with the second editor.